app.freelens.Freelens.metainfo.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>app.freelens.Freelens</id>
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+  <name>Freelens</name>
+  <developer_name>Freelens Authors</developer_name>
+  <url type="homepage">https://freelens.app</url>
+  <url type="help">https://github.com/flathub/app.freelens.Freelens?tab=readme-ov-file#usage</url>
+  <summary>FOSS UI for Kubernetes</summary>
+  <description>
+    <p>The Kubernetes management tool</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/freelensapp/freelens/refs/heads/main/.github/screenshot.png</image>
```

Comment:
don't use `main` branch use tag or commit.

Reply:
Ok, commit is used instead